packages/botonic-react/src/components/multichannel/multichannel-text.jsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react'
+import { RequestContext } from '../../contexts'
+import { Text } from '../text'
+import { Providers } from '@botonic/core'
+
+export class MultichannelText extends React.Component {
+  static contextType = RequestContext
+  constructor(props) {
+    super(props)
+    this.elements = []
+  }
+  render() {
+    if (
+      this.context.session &&
+      this.context.session.user &&
+      this.context.session.user.provider == Providers.Messaging.WHATSAPPNEW
+    ) {
+      let text = !Array.isArray(this.props.children)
```

Comment:
normally it's more clear to minimize the "nots":
       let text = Array.isArray(this.props.children) ? this.props.children.filter(e => !e.props) : [this.props.children]